README.md

```diff
@@ -1,4 +1,5 @@
 # Elasticsearch Rule
```

Comment:
Do the all comments in
 https://github.com/sahabpardaz/elasticsearch-rule/pull/11 
after merging that pull request.